debugger/HaxeServer.hx

```diff
@@ -139,8 +139,15 @@ class HaxeServer
             var peer = socket.peer();
             Sys.println("\nReceived connection from " + peer.host + ".");
 
-            HaxeProtocol.writeServerIdentification(socket.output);
-            HaxeProtocol.readClientIdentification(socket.input);
+            try {
```

Comment:
You've removed the throw of an exception when there is a protocol mismatch.  Are you sure that all code that uses this constructor is OK with silently ignoring this error (well, except for a new line printed to stdout, which may not even be presented to a user or otherwise noticed by the program)?  I'm not ...